includes/Core/Authentication/Authentication.php

```diff
@@ -408,12 +408,12 @@ private function handle_oauth() {
 				)
 			);
 
-			header( 'Location: ' . filter_var( $redirect_url, FILTER_SANITIZE_URL ) );
```

Comment:
Why remove this `filter_var()` call? That seems unrelated.

Reply:
In this case it's not necessary since the URL is from an internal source (`$this->context->admin_url()`). Also, WordPress core does not do this (which I found interesting) but I think it may be due to `wp_safe_redirect` calling `wp_sanitize_redirect` on the `$location` internally.

If we want to make it a policy to escape URLs in all redirects then I think we should standardize on using `esc_url_raw` for consistency.